packages/visx-grid/src/utils/polarToCartesian.ts

```diff
@@ -0,0 +1,16 @@
+export type PolarInput = {
```

Comment:
Would it be more generic to call these
`PolarCoordinate` and `CartesianCoordinate`?
in case somebody write a reverse transformation in the future the types can be reused.

Reply:
Makes sense. Renaming it as suggested